Library/src/Systems/Quota/Quota.cpp

```diff
@@ -231,118 +231,177 @@ void FeaturesQuotaResult::OnResponse(const csp::services::ApiResponseBase* ApiRe
 }
 const csp::common::String TierNameEnumToString(const TierNames& Value)
 {
+	csp::common::String TierString;
 	switch (Value)
 	{
 		case TierNames::Basic:
-			return "Basic";
+		{
+			TierString = "basic";
+			break;
+		}
 		case TierNames::Premium:
-			return "Premium";
+		{
+			TierString = "premium";
+			break;
+		}
 		case TierNames::Pro:
-			return "Pro";
+		{
+			TierString = "pro";
+			break;
+		}
 		case TierNames::Enterprise:
-			return "Enterprise";
+		{
+			TierString = "enterprise";
+			break;
+		}
+		default:
+		{
+			CSP_LOG_ERROR_FORMAT("Unknown quota tier encountered whilst converting the enum to string. Value passed in was %i. To fix this, add a "
+								 "case to the enum switch statement.",
+								 static_cast<int>(Value));
+			break;
+		}
 	}
+	return TierString;
 }
 
 const csp::common::String TierFeatureEnumToString(const TierFeatures& Value)
 {
+	csp::common::String TierString;
+
 	switch (Value)
 	{
 		case TierFeatures::Agora:
-			return "Agora";
+		{
+			TierString = "Agora";
+			break;
+		}
 		case TierFeatures::Shopify:
-			return "Shopify";
+		{
+			TierString = "Shopify";
+			break;
+		}
 		case TierFeatures::TicketedSpace:
-			return "TicketedSpace";
+		{
+			TierString = "TicketedSpace";
+			break;
+		}
 		case TierFeatures::AudioVideoUpload:
-			return "AudioVideoUpload";
+		{
+			TierString = "AudioVideoUpload";
+			break;
+		}
 		case TierFeatures::ObjectCaptureUpload:
-			return "ObjectCaptureUpload";
+		{
+			TierString = "ObjectCaptureUpload";
+			break;
+		}
 		case TierFeatures::OpenAI:
-			return "OpenAI";
+		{
+			TierString = "OpenAI";
+			break;
+		}
 		case TierFeatures::ScopeConcurrentUsers:
-			return "ScopeConcurrentUsers";
+		{
+			TierString = "ScopeConcurrentUsers";
+			break;
+		}
 		case TierFeatures::TotalUploadSizeInKilobytes:
-			return "TotalUploadSizeInKilobytes";
+		{
+			TierString = "TotalUploadSizeInKilobytes";
+			break;
+		}
 		case TierFeatures::SpaceOwner:
-			return "SpaceOwner";
+		{
+			TierString = "SpaceOwner";
+			break;
+		}
+		default:
+		{
+			CSP_LOG_ERROR_FORMAT("Unknown quota feature encountered whilst converting the enum to string. Value passed in was %i. To fix this, add a "
+								 "case to the enum switch statement.",
+								 static_cast<int>(Value));
+			break;
+		}
 	}
+
+	return TierString;
 }
 
 const TierNames StringToTierNameEnum(const csp::common::String& Value)
 {
+	TierNames TierName = TierNames::Basic;
+
 	if (Value == "basic")
 	{
-		return TierNames::Basic;
+		TierName = TierNames::Basic;
```

Comment:
Same here. Always prefer early returns over else-if.

Reply:
Addressed in latest.